sql/ashamane/world/2018_07_18_99_lightforge_spawns.sql

```diff
@@ -0,0 +1,290 @@
+INSERT INTO `gossip_menu` (`MenuId`, `TextId`) VALUES
+(22019, 33728), -- 130993
+(21910, 33495), -- 130986
+(21454, 32670), -- 126390
+(22008, 33709), -- 130560
+(22018, 33727), -- 132358
+(21423, 32623), -- 132224
+(21451, 32658), -- 126389
+(8522, 7028), -- 126022
+(21462, 9849), -- 123395
+(15145, 21709), -- 96507
+(9821, 13584), -- 96507
+(22017, 33725), -- 132334
+(22014, 33721); -- 132215
+
+DELETE FROM `gossip_menu_option` WHERE (`MenuId`=21910 AND `OptionIndex`=5) OR (`MenuId`=21910 AND `OptionIndex`=4) OR (`MenuId`=21910 AND `OptionIndex`=3) OR (`MenuId`=21910 AND `OptionIndex`=2) OR (`MenuId`=21910 AND `OptionIndex`=1) OR (`MenuId`=21910 AND `OptionIndex`=0) OR (`MenuId`=21423 AND `OptionIndex`=0) OR (`MenuId`=8522 AND `OptionIndex`=0) OR (`MenuId`=21462 AND `OptionIndex`=2) OR (`MenuId`=21462 AND `OptionIndex`=0) OR (`MenuId`=9821 AND `OptionIndex`=3) OR (`MenuId`=9821 AND `OptionIndex`=1);
+INSERT INTO `gossip_menu_option` (`MenuId`, `OptionIndex`, `OptionIcon`, `OptionText`, `OptionBroadcastTextId`) VALUES
+(21910, 5, 0, 'Can you teleport me back up to the bridge?', 0),
+(21910, 4, 0, 'How do you like being on the Vindicaar?', 0),
+(21910, 3, 0, 'Do you plan on staying with the draenei?', 0),
+(21910, 2, 0, 'What do you know about becoming Lightforged?', 0),
+(21910, 1, 0, 'I passed my trial!', 0),
+(21910, 0, 0, 'Hello, Vigilant Quoram. We met before on Argus.', 0),
+(21423, 0, 1, 'I need repairs, Durael.', 0),
+(8522, 0, 3, 'Train me.', 0),
+(21462, 2, 1, 'Let me browse your goods.', 0),
+(21462, 0, 5, 'Make this inn your home.', 0),
+(9821, 3, 0, 'I''d like to heal and revive my battle pets.', 0),
+(9821, 1, 1, 'I''m looking for a lost companion.', 0);
+
+/*  Needs Scripting?
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `comment`) VALUES
+(130986, @GROUP_ID+0, @ID+, 'I am afraid I cannot allow that.', 12, 0, 100, 0, 0, 96050, UNKNOWN, 'Vigilant Quoram to Player'),
+(130986, @GROUP_ID+1, @ID+, 'Fine.', 12, 0, 100, 0, 0, 96049, UNKNOWN, 'Vigilant Quoram to Player'),
+(130986, @GROUP_ID+2, @ID+, 'For now.', 12, 0, 100, 0, 0, 96047, UNKNOWN, 'Vigilant Quoram to Player'),
+(130986, @GROUP_ID+3, @ID+, 'Congratulations.', 12, 0, 100, 0, 0, 96043, UNKNOWN, 'Vigilant Quoram to Player'),
+(130986, @GROUP_ID+4, @ID+, 'Why have you returned?', 12, 0, 100, 0, 0, 96042, UNKNOWN, 'Vigilant Quoram to Player'),
+(130986, @GROUP_ID+5, @ID+, 'It is designed to kill you.', 12, 0, 100, 0, 0, 96044, UNKNOWN, 'Vigilant Quoram to Player'),
```

Comment:
Basic SAI will help for Vigilant Quoram texts. Since they're a response from some gossip options
https://www.youtube.com/watch?v=C0umFXQlYTs (it's hilarious)